python_modules/libraries/dagster-dbt/dagster_dbt/types.py

```diff
@@ -112,7 +112,7 @@ def from_dict(cls, d: Dict[str, Any]) -> "NodeResult":
             node = check.dict_elem(d, "node")
             unique_id = check.str_elem(node, "unique_id")
         error = check.opt_str_elem(d, "error")
-        execution_time = check.float_elem(d, "execution_time")
+        execution_time = d["execution_time"]
```

Comment:
nit: we should still check here; can check multiple types by doing `check.inst_param(d["execution_time"], "execution_time", (int, float))`.